project_event/models/project_project.py

```diff
@@ -240,3 +240,26 @@ def map_tasks(self, new_project_id):
                 'project_id': new_project_id}
             tasks += task.copy(defaults)
         return self.browse(new_project_id)
+
+    @api.multi
+    def _message_track(self, tracked_fields, initial):
+        mail_track = super()._message_track(tracked_fields, initial)
+        changes = mail_track[0]
+        tracking_value_ids = mail_track[1]
+        order_fields = self.order_event_fields(tracking_value_ids)
+        return changes, order_fields
+
+    @staticmethod
+    def order_event_fields(tracking_values):
+        event_fields_list = [
```

Comment:
Field order: 'state',  'name', 'code', 'responsible_id', 'partner_id', 'notes'

Reply:
:+1:

Reply:
though, user_id must also be here

Reply:
chatter doesn't record any new modifications (only track the fields of the new created event)

Reply:
I check this and work well in my local, and the tests pass, @lgharib does it work in you machine?
Thanks